fili-core/src/main/java/com/yahoo/bard/webservice/druid/serializers/Util.java

```diff
@@ -44,4 +45,25 @@
         // If we cannot find the physical name, then return empty optional.
         return Optional.empty();
     }
+
+    /**
+     * JSON tree walk to determine if there is a nested query below the current json node or not.
+     *
+     * @param gen  the Json Generator to retrieve the tree to walk on
+     *
+     * @return  a Boolean where true indicates there are more nested query below this node, false otherwise
+     */
+    public static Boolean moreNestedQuery(JsonGenerator gen) {
```

Comment:
Rather than "more", for methods that are essentially "property getters", common convention is to use "has" or "is" as the method prefix. So, in this case, it would be "hasNestedQuery". (or `hasInnerQuery`)